packages/builder/src/fee/FeeBuilder.utils.ts

```diff
@@ -0,0 +1,247 @@
+/* eslint-disable sort-keys */
+import { AnyParachain, ChainAssetId } from '@moonbeam-network/xcm-types';
+import { ApiPromise } from '@polkadot/api';
+import { Option, Result, u128 } from '@polkadot/types';
+import { Error as PolkadotError, Weight } from '@polkadot/types/interfaces';
+import { AnyJson } from '@polkadot/types/types';
+import { XcmVersion } from '../extrinsic';
+import { MoonbeamRuntimeXcmConfigAssetType } from './FeeBuilder.interfaces';
+
+const DEFAULT_AMOUNT = 10 ** 6;
+const DEFALUT_HEX_STRING =
+  '0xabcdef1234567890fedcba0987654321abcdef1234567890fedcba0987654321';
+
+const moonChainNativeAssetId = '0x0000000000000000000000000000000000000802';
+
+const moonChainBalancesPalletInstance: Record<string, string> = {
+  moonbeam: '10',
+  moonriver: '10',
+  'moonbase-alpha': '3',
+};
+const moonChainERC20PalletPalletInstance: Record<string, string> = {
+  moonbeam: '110',
+  moonriver: '110',
+  'moonbase-alpha': '48',
+};
+
+const XCM_VERSION: XcmVersion = XcmVersion.v4; // TODO
+
+function isXcmV4() {
+  return XCM_VERSION === XcmVersion.v4;
+}
```

Comment:
Looks like this does nothing.

Reply:
Right now it always returns true because the version is hardcoded to V4. I wanted to leave it programmed in case we need the format of V3 or earlier, but it can be removed as well